app/i18n/tr/gen.php

```diff
@@ -111,7 +111,7 @@
 		'mar' => 'mar',
 		'march' => 'Mar',
 		'may' => 'Mayıs',
-		'may_' => 'May',	// IGNORE
+		'may_' => 'May',
```

Comment:
```suggestion
		'may_' => 'May',	// IGNORE
```

Reply:
When I looked at other language files, the // IGNORE field was removed in some of them. I guess there is no change between Turkish and English, so // IGNORE was added.

I thought it could be removed. @Alkarex